src/main/java/com/cdw/gatekeeper/dto/SuccessResponseDTO.java

```diff
@@ -0,0 +1,12 @@
+package com.cdw.gatekeeper.dto;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+/**
+ * DTO for success responses
+ */
+@Data
+public class SuccessResponseDTO {
+    private HttpStatus httpStatus;
+    private Object body;
```

Comment:
specifying object type is not recommended but considering this is purely backend app we can leave it.